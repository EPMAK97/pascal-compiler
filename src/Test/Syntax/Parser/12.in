var x : integer;
Begin
    x := 1;
    if x > 10 then
        x := 24;
end.
