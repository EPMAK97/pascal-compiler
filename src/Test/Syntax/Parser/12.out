└── Main Program
    ├── var
    │   └── integer
    │       └── x
    ├── x
    │   └── 1
    └── if
        ├── >
        │   ├── x
        │   └── 10
        └── x
            └── 24
